spec.html

```diff
@@ -33069,7 +33075,7 @@ <h1>SystemTimeZoneIdentifier ( ): a String</h1>
         <h1>
           LocalTime (
             _t_: a finite time value,
-          ): an integral Number
+          ): a local-offset time value
```

Comment:
If, for some reason, we *don't* make all local-offset time values finite, we should add the finite qualifier here.